src/script/components/TitleBar/index.test.tsx

```diff
@@ -137,6 +145,8 @@ describe('TitleBar', () => {
     const panelViewModel = createPanelViewModel();
     const conversation = createConversationEntity();
 
+    mockedUiKit.useMatchMedia.mockReturnValue(false);
```

Comment:
challenge for you: write a test that checks that the component renders correctly when the view is scaledDown 😉